hudi-common/src/main/java/org/apache/hudi/common/table/view/IncrementalTimelineSyncFileSystemView.java

```diff
@@ -123,6 +123,7 @@ private void runIncrementalSync(HoodieTimeline timeline, TimelineDiffResult diff
 
     LOG.info("Timeline Diff Result is :" + diffResult);
 
+    viewLock.hasWriteLock();
```

Comment:
Is this line necessary if we're not checking the return value?

Reply:
My mistake, assertion has been added